lib/Page.js

```diff
@@ -694,6 +694,43 @@ class Page extends EventEmitter {
   }
 
   /**
+   * @param {string} selector
+   * @param {string|string[]} value
+   */
+  async select(selector, value) {
+    const handle = await this.$(selector);
+    console.assert(handle, 'No node found for selector: ' + selector);
+
+    await this.evaluate((element, value) => {
+      if (element.nodeName.toLowerCase() !== 'select')
+        return false;
+
+      if (Array.isArray(value) !== true)
+        value = [value];
+
+      if (element.multiple !== true) {
+        element.value = value[0];
+
+        if (element.value !== value[0])
+          [].some.call(element.options, option => option.selected = value[0] === option.label);
```

Comment:
let's convert options to array once to avoid the `Array.prototype.call` workaround:

```js
const options = Array.from(element.options);
// ...
```